client/packages/system/src/Item/DetailView/Tabs/MasterLists.tsx

```diff
@@ -13,26 +13,29 @@ import {
   createQueryParamsStore,
   TooltipTextCell,
   useTranslation,
+  useAuthContext,
 } from '@openmsupply-client/common';
 
 const MasterListsTable = ({ itemId }: { itemId?: string }) => {
+  const t = useTranslation();
+  const { store } = useAuthContext();
```

Comment:
Side Quest: Move `useTranslation` hook to the top. It's an import hook, but not essential for understanding the component’s main business logic